pages-includes/docs/tutorials/native/basic-kotlin-native-app-codeblocks-code.md

```diff
@@ -20,7 +20,7 @@ kotlin {
 }
 
 wrapper {
-  gradleVersion = "5.3.1"
+  gradleVersion = "6.7"
```

Comment:
There is a placeholder for Gradle version as well -  `{{ site.data.releases.tutorials.native.gradle_version }}`
However, it's now outdated. Please change the version here and update the placeholder in [/data/releases.yml](https://github.com/JetBrains/kotlin-web-site/blob/bd50bea578ed517765d18876d52accc93bd8d237/data/releases.yml#L25)